prac_09/files_to_sort_v2.py

```diff
@@ -0,0 +1,32 @@
+import os
+import shutil
+
+
+def main():
+    # Move to desired directory
+    os.chdir('FilesToSort')
+    print("Current directory is", os.getcwd())
+
+    for dir_name, subdir_list, file_list in os.walk('.'):
```

Comment:
We don't need to do this for subfolders - just the files in one folder (so don't use walk).